packages/web-api/src/response/ViewsPublishResponse.ts

```diff
@@ -2,12 +2,141 @@
 import { WebAPICallResult } from '../WebClient';
 export type ViewsPublishResponse = WebAPICallResult & {
   ok?:                boolean;
+  warning?:           string;
   error?:             string;
-  response_metadata?: ResponseMetadata;
   needed?:            string;
   provided?:          string;
+  view?:              View;
+  response_metadata?: ResponseMetadata;
 };
 
 export interface ResponseMetadata {
   messages?: string[];
 }
+
+export interface View {
+  id?:                    string;
```

Comment:
also, is there a reason why every single property is optional, even though this is not the case? `id` for example should always be present

Reply:
the same as my above comment